client/src/Components/CommonComponent/Header/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Menu from '../Menu';
+import logo from '../../../assets/logo.png';
+
+import('./index.css');
+
+class Header extends Component {
+  state = {
+    show: false,
+    isSigned: false,
+  };
+
+  onClickHandler = () => {
+    this.setState(oldState => ({
+      show: !oldState.show,
+    }));
+  };
+
+  render() {
+    const { show, isSigned } = this.state;
+    return (
+      <React.Fragment>
+        <header className="App-header">
+          <div className="logo">
+            <img src={logo} alt="Site Logo" className="site-logo" />
+          </div>
+
+          <div className="App-identifier">Parent assistant</div>
+          <Link onClick={this.onClickHandler} className="hamburger" to="/">
```

Comment:
The name `onClickHandler` does not say much. A better name would be `toggleMenu`, `showHideMenu`, or sth similar; so that when you read this line you would instantly know what clicking on the Link does..